lib/tasks/jobs/data.rake

```diff
@@ -0,0 +1,8 @@
+namespace :job do
+  namespace :data do
+    desc "Remove data that is old enough that it should be removed"
+    task clear: :environment do
```

Comment:
question - just checking that we need` :environment ` so that we can access all our models for this rake task - was getting confusing google search results.

Reply:
Exactly. As I understand it we're saying this task inherits from the `environment` task which contains logic to include all our models and stuff so it's available to be called within the body of the task.